x/cscalist/keeper/proposal.go

```diff
@@ -0,0 +1,24 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/rarimo/rarimo-core/x/cscalist/types"
+)
+
+func (k Keeper) EditCSCAListProposal(ctx sdk.Context, proposal *types.EditCSCAListProposal) error {
```

Comment:
why rebuild the whole tree? we can just add/remove a node

Reply:
@violog

Reply:
implementation of second proposal in progress